src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,336 @@
+
+```
+const transport = new QuicTransport(QUIC_URL);
+await transport.ready;
+
+const ws = transport.sendDatagrams();
+const stream = await transport.createSendStream();
+```
+
+QuicTransport is a valid alternative to WebSockets, as it supports the use cases from the WebSocket API and adds support for scenarios where minimal latency is more important than reliability and message order. This makes it a good choice for games and applications dealing with high-frequency events.
+
+{{ figure_markup(
+  image="quic_transport.png",
+  alt="Percentage of page loads in Chrome using QuicTransport",
+  caption='Percentage of page loads in Chrome using QuicTransport. (Source: <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3184">Chrome Platform Status metrics</a>)',
+  description="Chart of QuicTransport usage, based on the percentage of page loads in Chrome using this feature. It shows a peak in October 2020, where the API was used during approximately 0.00089 percent pf page loads.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1571330893&format=interactive",
+  sheets_gid="708893754"
+  )
+}}
+
+The use of the interface is currently still so low that it's hardly measurable. In October 2020, it has increased strongly and is now used during 0.00089 percent of page loads in Chrome.
+
+## Conclusion
+
+The state of web capabilities in 2020 is healthy, as new powerful APIs regularly ship with new releases of Chromium-based browsers. Some interfaces like the Content Indexing API or Idle Detection API help to add finishing touches to certain web applications. Other APIs, such as the File System Access and Async Clipboard API, allow a whole new application category, namely productivity apps, to finally fully make the shift to the web. Some APIs such as Async Clipboard and Web Share API have already made their way into other, non-Chromium browsers. Safari even was the first mobile browser to implement the Web Share API.
+
+Through its [rigorous process](https://developers.google.com/web/updates/capabilities#process), the Fugu team ensures that access to these features takes place in a secure and privacy-friendly manner. Additionally, the Fugu team actively solicits the feedback from other browser vendors and web developers. While the usage of most of these new APIs is comparatively low, some APIs presented in this chapter show an exponential or even hockey stick-like growth, such as the Badging or Content Indexing API. It is dependent on web developers what the state of web capabilities in 2021 will look like: The author encourages the community to build great web applications, make use of the powerful APIs in a backwards-compatible manner, and help make the web a more capable platform.
```

Comment:
Could we include a link for the place where developers can give feedback on these APIs?